client/state/preferences/README.md

```diff
@@ -45,5 +45,6 @@ To add a new preference key, you only need to add a key and defaut value in `DEF
 ### Actions
 
 - **fetchPreferences()** - populate the tree
-- **setPreference( key, value )** - set preference and persist all preferences in the endpoint
+- **savePreference( key, value )** - set preference and persist all preferences in the endpoint
```

Comment:
Ah, 👍  I see that you're explicitly excluding mediaModalGalleryInstructionsDismissedForSession from being persisted.